cmd/buck/cli/archive.go

```diff
@@ -197,25 +211,104 @@ var archiveCmd = &cobra.Command{
 
 		var opts []local.ArchiveRemoteOption
 
+		config := local.ArchiveConfig{}
 		if reader != nil {
 			buf := new(bytes.Buffer)
 			_, err := buf.ReadFrom(reader)
 			cmd.ErrCheck(err)
 
-			config := local.ArchiveConfig{}
 			cmd.ErrCheck(json.Unmarshal(buf.Bytes(), &config))
 
-			opts = append(opts, local.WithArchiveConfig(config))
+		} else {
+			config, err = buck.DefaultArchiveConfig(ctx)
+			cmd.ErrCheck(err)
 		}
+		opts = append(opts, local.WithArchiveConfig(config))
 
-		conf, err := bucks.NewConfigFromCmd(c, ".")
+		addrs, err := buck.Addresses(ctx)
 		cmd.ErrCheck(err)
-		ctx, cancel := context.WithTimeout(context.Background(), cmd.Timeout)
-		defer cancel()
-		buck, err := bucks.GetLocalBucket(ctx, conf)
+		if len(addrs.Addresses) != 1 {
+			cmd.Error(fmt.Errorf("There should be exactly one wallet address but are there're %d", len(addrs.Addresses)))
```

Comment:
"There should be exactly one wallet address but there are %d"

Reply:
Oops, thanks!